interfaces/apparmor/apparmor.go

```diff
@@ -49,6 +52,11 @@ func LoadProfile(fname string) error {
 	return nil
 }
 
+// RemoveCachedProfile removes binary cache file from /var/cache/apparmor
+func RemoveCachedProfile(profile string) error {
+	return os.Remove(filepath.Join(dirs.AppArmorCacheDir, profile))
```

Comment:
Why isn't that being done inside UnloadProfile itself?  This would make it the complement of LoadProfile, which writes the cache.